### DISCUSSION TITLE
How can i do: Central columns wider

### DISCUSSION THREAD

**Discussion by alg0r1txm** [thuliteio/doks#933] [Q&A]
How can I change the proportion of the columns in the docs, for example the content column to be wider than the default one.

![sample](https://user-images.githubusercontent.com/66405379/198882869-5add0dc0-84f1-42e9-b014-8804c5d376ec.png)

**Comment by h-enk** [comment]
Set `fullWidth` to `true`: https://github.com/h-enk/doks/blob/master/config/_default/params.toml#L106

**Reply by toastedqu** [reply]
Hi, how may I do that with the newest version? This param does not exist anymore.

**Reply by h-enk** [reply]
Set:

```toml
containerBreakpoint = "fluid"
```

https://github.com/gethyas/doks/blob/main/config/_default/params.toml#L20
